xradar/accessors.py

```diff
@@ -204,3 +219,25 @@ def add_crs(self) -> dt.DataTree:
         """
         ds = self.xarray_obj
         return ds.pipe(add_crs_tree)
+
+    def apply(self, func, pass_sweep_name=False, *args, **kwargs):
```

Comment:
We should keep the signature as clean as possible. We always can extract `pass_sweep_name` from *args or **kwargs.

Reply:
Agreed, I like your suggesstion